config/rules.yaml

```diff
@@ -35,3 +35,7 @@ rules:
   - match: udp
     type: conn_handler
     target: udp
+  - match: tcp dst port 3260
```

Comment:
The rules are executed top to bottom, you want to place this port-specific rule before the generic TCP rule.